src/templates/settings.html

```diff
@@ -0,0 +1,152 @@
+                </tr>
+                <tr class="admin" style="display: none"><td>&nbsp</td></tr>
+                <tr class="admin" style="display: none">
+                    <td width="35%"><b>Change Status to Admin</b></td>
+                    <td width="15%"><input id="adminEmail" type="text" placeholder="User's Email" /></td>
+                    <td width="15%">&nbsp</td>
+                    <td width="15%"><button id="submitAddAdmin" type="button" class="">Change</button> </td>
+                    <td id="addAdminNotification" width="20%" style="color:red; visibility: hidden">  *Incorrect</td>
+                </tr>
+                <tr class="admin" style="display: none"><td>&nbsp</td></tr>
+                <tr class="admin" style="display: none">
+                    <td width="35%"><b>Remove Author</b></td>
+                    <td width="15%"><input id="removalAuthorEmail" type="text" placeholder="User's Email" /></td>
+                    <td width="15%">&nbsp</td>
+                    <td width="15%"><button id="submitRemoveAuthor" type="button" class="">Remove</button> </td>
+                    <td id="removeAuthorNotification" width="20%" style="color:red; visibility: hidden">  *Incorrect</td>
+                </tr>
+                <tr class="admin" style="display: none"><td>&nbsp</td></tr>
+                <tr class="admin" style="display: none">
+                    <td width="35%"><b>Remove Admin</b></td>
+                    <td width="15%"><input id="removalAdminEmail" type="text" placeholder="User's Email" /></td>
+                    <td width="15%">&nbsp</td>
+                    <td width="15%"><button id="submitRemoveAdmin" type="button" class="">Remove</button> </td>
+                    <td id="removeAdminNotification" width="20%" style="color:red; visibility: hidden">  *Incorrect</td>
+                </tr>
+            </table>
+        </a>
+    </div>
+</div>
+
+{% endblock %}
+{% block js_bottom %}
+{{ super() }}
+<script>    
+    function addAuthor() {
+        $.ajax({
+              url: '{{ url_for("add_Author") }}',
+              type: 'POST',
+              contentType: 'application/json',
+              data: JSON.stringify({
+                    'email': $("#authorEmail").val(),
+                }),
+                success:  function(data) {
+                    if(data == "failure"){
+                        $("#addAuthorNotification").css("visibility","visible");
+                    }else{
+                        alert(data + " is now an Author");
+                        $("#addAuthorNotification").css("visibility", "hidden");
+                        $("#addAuthorEmail").val("");
+
+                    }
+                },
+                error: function(data){    
+                }
+        });
+    }
+
+    function addAdmin() {
+        var code = prompt("Please enter this course's Security Code", "123456");
```

Comment:
I'm confused by this.  Why does the prompt say enter the course's security code? Isn't this supposed to be the security code for creating admins that only Nir will know?